packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -32,12 +33,6 @@ export class VaFileInput {
   private fileInputRef!: HTMLInputElement;
   private uploadStatus: 'idle' | 'success' = 'idle';
   private fileType?: string;
-  private defaultUploadMessage: HTMLElement = (
-    <span>
-      Drag a file here or{' '}
-      <span class="file-input-choose-text">choose from folder</span>
-    </span>
-  )
```

Comment:
moved this variable into a function because when calling `forceUpdate` after reaching the end of upload Stencil was throwing errors relating to caching the JSX.